src/state/auth/tokenRefresh/saga.js

```diff
@@ -0,0 +1,59 @@
+import {
+  call,
+  put,
+  select,
+  takeLatest,
+} from 'redux-saga/effects'
+
+import { dispatchAction, makeFetchCall } from 'state/utils'
+import { updateAuth } from 'state/auth/actions'
+import { updateErrorStatus } from 'state/status/actions'
+import { getAuthData } from 'state/auth/selectors'
+
+import types from './constants'
+import actions from './actions'
+
+const REFRESH_INTERVAL = 5 * 60 * 1000
+let interval = null
```

Comment:
I think the interval reference can be in the store so no side effect here, but it is ok.